webui/react/package.json

```diff
@@ -8,8 +8,8 @@
   },
   "scripts": {
     "postshrinkwrap": "npx npm-force-resolutions && npm install --ignore-scripts",
-    "start": "craco start",
-    "build": "craco build",
+    "start": "vite serve",
+    "build": "tsc; NODE_OPTIONS=--max-old-space-size=6144 vite build --mode=${DET_NODE_ENV:-production}",
```

Comment:
Yes -- the bundler that vite uses is less well behaved than webpack when it comes to memory usage. We're using 6gb in particular because our ci container allocates 8gb -- while the program should gracefully allocate extra memory to disk if `max-old-space` exceeds the physical memory of the container, that's slow.